### ISSUE TITLE
local import "../json-dns" in non-local package

### ISSUE THREAD

**Issue by matriphe** [m13253/dns-over-https#6]
Try by downloading this package using `go get github.com/m13253/dns-over-https`.

I'm on MacOSX 10.13.3 and using Go 1.10.

```
mactriphe:src matriphe$ go get github.com/m13253/dns-over-https
package github.com/m13253/dns-over-https: no Go files in /Volumes/data/Development/golang/src/github.com/m13253/dns-over-https
mactriphe:src matriphe$ cd github.com/m13253/dns-over-https/
mactriphe:dns-over-https matriphe$ make
go get -d -v ./doh-client ./doh-server
github.com/miekg/dns (download)
github.com/BurntSushi/toml (download)
github.com/gorilla/handlers (download)
doh-client/client.go:37:2: local import "../json-dns" in non-local package
doh-server/google.go:36:2: local import "../json-dns" in non-local package
make: *** [deps] Error 1
mactriphe:dns-over-https matriphe$
```

**Comment by m13253** [comment]
I guess I can fix it with 0a107be362a8790917e02d565da0b2ba8e7b5016.
Would you please try that again?

The reason is that, this repository contains 3 packages: `doh-client`, `doh-server`, `json-dns`.
The common `json-dns` is causing problems.

It should work now. Thank you for your contribution.

**Comment by matriphe** [comment]
Thanks for the fix! Now it's working! 👍
